src/migrations/20200805091409-add-feature-toggle-type.js

```diff
@@ -21,11 +21,11 @@ exports.up = function (db, cb) {
             db.runSql.bind(
                 db,
                 `
-              INSERT INTO feature_types(id, name, description, lifetime_days) VALUES('release', 'Release', 'Used to enable trunk-based development for teams practicing Continuous Delivery.', 40);
-              INSERT INTO feature_types(id, name, description, lifetime_days) VALUES('experiment', 'Experiment', 'Used to perform multivariate or A/B testing.', 40);
-              INSERT INTO feature_types(id, name, description, lifetime_days) VALUES('operational', 'Operational', 'Used to control operational aspects of the system behavior.', 7);
-              INSERT INTO feature_types(id, name, description, lifetime_days) VALUES('kill-switch', 'Kill switch', 'Used to to gracefully degrade system functionality.', null);
-              INSERT INTO feature_types(id, name, description, lifetime_days) VALUES('permission', 'Permission', 'Used to change the features or product experience that certain users receive.', null);
+                INSERT INTO feature_types(id, name, description, lifetime_days) VALUES('release', 'Release', 'Release feature toggles are used to release new features.', 40);
+                INSERT INTO feature_types(id, name, description, lifetime_days) VALUES('experiment', 'Experiment', 'Experiment feature toggles are used to test and verify multiple different versions of a feature.', 40);
+                INSERT INTO feature_types(id, name, description, lifetime_days) VALUES('operational', 'Operational', 'Operational feature toggles are used to control aspects of a rollout.', 7);
+                INSERT INTO feature_types(id, name, description, lifetime_days) VALUES('kill-switch', 'Kill switch', 'Kill switch feature toggles are used to quickly turn on or off critical functionality in your system.', null);
```

Comment:
This will not work! You cannot change an existing migration, this will only affect fresh installations, but have no effect on existing, as a migration will only be executed once. 

You will have to create a new migration. You can use the db-migrate scripts to create a new migration file:
https://github.com/Unleash/unleash/blob/master/website/docs/contributing/developer-guide.md#database-changes-database-changes

Reply:
@ykhedher

Reply:
I didn't see that. I will create a new migration then ;)